report/assets/templates.html

```diff
@@ -562,6 +562,18 @@
   </a>
 </template>
 
+<!-- Lighthouse category ratio -->
+<template id="ratio">
```

Comment:
I've been calling this "ratio", but I've seen others refer to it as "fraction". Any strong opinions about the name?

Reply:
I typically refer to a representation like this as a "fraction" when the first number is strictly less than or equal to the second number, and a "ratio" when the two numbers have a more arbitrary relationship.

By that lens, I would call these a fraction.